cockroachdb/values.yaml

```diff
@@ -228,6 +228,9 @@ statefulset:
     #   cpu: 100m
     #   memory: 512Mi
 
+  customLivenessProbe: {}
+  customReadinessProbe: {}
```

Comment:
Can you add these values to [cockroachdb/README.md#Configuration](https://github.com/cockroachdb/helm-charts/blob/master/cockroachdb/README.md#configuration), and provide an example here in values.yaml of what a customLivenessProbe might look like?